Dolapo47/Auto-Mart
# Ft new sale #165951358

#### What does this PR do?
- A summary of what this task is about
Create a page where the user can make a new car post

#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
-Create the header
-Create a form to accept the car information
-create a submit button to post a new car  

#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
- Fill in the input field with the necessary information
- Submit the form for a new post

#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#165951358](https://www.pivotaltracker.com/n/projects/2346179)`)
